README.md

```diff
@@ -86,21 +95,34 @@ time.sleep(1) # allow the messages to send
 while relay_manager.message_pool.has_events():
   event_msg = relay_manager.message_pool.get_event()
   print(event_msg.event.content)
-  
+
 relay_manager.close_connections()
 ```
 
 ## Installation
+
 ```bash
 pip install nostr
 ```
 
 Note: I wrote this with Python 3.9.5.
 
-## Test Suite
-See the [Test Suite README](test/README.md)
+## Running tests
+
+Install tox
+
+```
+pip install tox
+```
+
+Run tests
+
+```
+tox
+```
```

Comment:
Is this related to this PR? Seems like it should be a separate PR and discussed separately since you're changing the testing framework.